docs/tendermint-core/configuration.md

```diff
@@ -127,6 +127,15 @@ max_subscriptions_per_client = 5
 # See https://github.com/tendermint/tendermint/issues/3435
 timeout_broadcast_tx_commit = "10s"
 
+# The name of cert file that used to serve RPC based on Tls.
+# NOTE: useful when tls_key_file is present. If not tls_cert_file
+# and tls_key_file both present, will use RPC based on HTTP.
+tls_cert_file = ""
+
+# The name of private key file that used to serve RPC based on Tls.
```

Comment:
```
# The name of a file containing matching private key that is used to create the HTTPS server.
# NOTE: both tls_cert_file and tls_key_file must be present for Tendermint to create HTTPS server. Otherwise, HTTP server is run.
```

Reply:
done